Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.py

```diff
@@ -4705,6 +4993,268 @@ def get_papi_property_rule_command(client: Client,
         return human_readable, {}, {}
 
 
+# Created by D.S. 2022-11-25
+def get_papi_property_byname_command(client: Client,
+                                     contract_id: str,
+                                     group_id: str,
+                                     property_name: str,) -> Tuple[str, Dict[str, Any], Union[List, Dict]]:
+    """
+        Get papi property within a group by property name
+    Args:
+        client: Client object with request
+        contract_id: Unique identifier for the contract
+        property_name: name of the property
+        group_id: Unique identifier for the group
+
+    Returns:
+        human readable (markdown format), entry context and raw response
+    """
+
+    raw_response: Dict = client.list_papi_property_bygroup(contract_id=contract_id,
+                                                           group_id=group_id,
+                                                           )
+    lookupKey = 'propertyName'
+    lookupValue = property_name
+    returnDict = next((item for item in raw_response["properties"]["items"]
+                       if item[lookupKey] == lookupValue), None)
+    if returnDict is not None:
+        raw_response = client.get_papi_property_bygroup(contract_id=contract_id,
+                                                        group_id=group_id,
+                                                        property_id=returnDict['propertyId'],
+                                                        )
+        title = f'{INTEGRATION_NAME} - get papi property byname command'
+        entry_context, human_readable_ec = get_papi_property_bygroup_ec(raw_response['properties']['items'][0])
+        context_entry: Dict = {
+            f"{INTEGRATION_CONTEXT_NAME}.PapiProperty.Found": entry_context
+        }
+        human_readable = tableToMarkdown(
+            name=title,
+            t=human_readable_ec,
+            removeNull=True,
+        )
+        return human_readable, context_entry, raw_response
+    else:
+
+        err_msg = f'{INTEGRATION_NAME} - get papi property command: Property {property_name} is not found'
+        raise DemistoException(err_msg)
+        # human_readable = f'{INTEGRATION_NAME} - get papi property command: Property {property_name} is not found'
+        # return human_readable, {}, {}
+
+
+# Created by D.S. 2022-11-25
```

Comment:
Please remove comment

Reply:
We have two individuals working on the same integration and only using the Xsoar's Web based development interface.
We are planning to get the CI/CD pipeline setup within this year. However, before that is ready, we are probably going to keep track of some of our work in this manner.